### PR TITLE
Add protobuf dependency in requirements

### PR DESCRIPTION
**Pull request by sanjgupt**

### CODE REVIEW

File: azure-quantum/requirements.txt [hunk 1]
```diff
@@ -7,3 +7,4 @@ numpy>=1.21.0
 deprecated>=1.2.12
 aiohttp>=3.7.0
 aiofile>=3.7.2
+protobuf==3.14.0
```
Comment (guenp) [inline azure-quantum/requirements.txt:10]:
```suggestion
protobuf>=3.14.0
```

**Review by guenp — CHANGES_REQUESTED** [review]
Can we use this version as a lower limit?

**Review by guenp — APPROVED** [review]

**Review by anpaz — APPROVED** [review]

**Comment by guenp** [comment]
Looks like we've got conflicting version numbers:
> The conflict is caused by:
    azure-quantum[cirq,qiskit] 0.21.2111.35696a1 depends on protobuf>=3.14.0
    cirq-google 0.12.0 depends on protobuf~=3.13.0

**Comment by guenp** [comment]
@sanjgupt I've pushed a change to drop `cirq-google` from the requirements (since we really only need `cirq-core`)

**Pull request merged**